### PR TITLE
Adding database BridgeDb structure (SQL) and documentation folder to Git Repo

### PR DESCRIPTION
**Pull request by nunogit**

### CODE REVIEW

File: docs/database/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# BridgeDb database structure
+
+These creation statements were extracted from a Derby file.
+They may not correspond exactly to the original structure.
```
Comment (egonw) [inline docs/database/README.md]: Can you clarify here if this SQL also applies to the MySQL backend?

**Comment by egonw** [comment]
Good starting point. The README about the database structure is pretty empty (it leaves many questions). I left one question.

File: docs/database/README.md (hunk 1 above)
Comment (nunogit) [reply docs/database/README.md]: It does work, I removed the indexes though.

Comment (egonw) [reply docs/database/README.md]: What do you mean with "it does work"? So, this schema is identical for Derby and MySQL?

Comment (nunogit) [reply docs/database/README.md]: I can't tell if it's similar, in the sense that I don't have the original one.
This schema **is** compatible with MySQL. I tested it. I removed the indexes though. So I didn't test with the indexes.

**Pull request merged**